.github/CONTRIBUTING.md

```diff
@@ -8,11 +8,9 @@ see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq
 
 ## Getting Started
 
-This library is relatively easy to build. To build and test this locally, 
-make sure you've done the following:
+To build and test this locally, make sure you've done the following:
 - Java 1.8 SDK is installed
-- Maven 3.x is installed
-- Scala 2.11.X is installed
+- Maven 3.x is installed (or SBT version 0.13.13)
```

Comment:
I'd fix the version to "1.1.1" since this is the latest supported one. People could use earlier versions on their own, but the official docs (since we're at it) should promote the latest and greatest if it just works.

I also thought about linking the tools to their websites (perhaps even the download sections), but that could be just nitpicking.

Reply:
Updated and links added 👍